File: despatch.WEB/app/src/swagger/models/contact-data-phone-numbers-response.ts
/* tslint:disable */
import { BaseEntityResponse } from './base-entity-response';
import { ContactDataResponse } from './contact-data-response';
import { PhoneNumberResponse } from './phone-number-response';
export interface ContactDataPhoneNumbersResponse extends BaseEntityResponse{
  contactData?: ContactDataResponse;
  contactDataId?: string;
  phoneNumber?: PhoneNumberResponse;
  phoneNumberId?: string;
}

File: despatch.Domain/Models/Default/Response/ContactDataPhoneNumbersResponse.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace despatch.Domain.Models.Default.Response
{
    public class ContactDataPhoneNumbersResponse : BaseEntityResponse
    {
        public string ContactDataId { get; set; }
        public string PhoneNumberId { get; set; }

        public virtual ContactDataResponse ContactData { get; set; }
        public virtual PhoneNumberResponse PhoneNumber { get; set; }
    }
}
